src/tests-environments/meteor.ts

```diff
@@ -64,6 +66,46 @@ export class MeteorStressTest {
       this.subscriptions[subscriptionName] = result;
     });
   }
+  
+  call(methodName: string, ...args): Promise<any> {
+    return new Promise((resolve, reject) => {
+      this.ddpClient.call(methodName, args, (err, result) => {
+        if (err) {
+          reject(err);
+        }
+        else {
+          resolve(result);
+        }
+      });
+    });
+  }
+  
+  login(username: string, email: string, password: string): Promise<any> {
+    const user = this.getValidUser(email, username);
+    return this.call('login', {
+      user,
+      password: this.hashPassword(password)
+    });
+  }
+  
+  getValidUser(email: string, username: string): Object {
```

Comment:
Maybe instead of `Object`, you can return a custom type/interface that have two optional fields - email/username